packs/smart_items/assets/video_player/data.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "dc5ec17b-aa1a-4bd0-bace-7d615d9bdc70",
   "name": "Video Player",
-  "category": "media",
+  "category": "deprecated",
```

Comment:
as it will be deprecated, what do you think about removing the whole item from here?

the scenes that already contain the Video Player will continue working without issues